app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command) != 3:
```

Comment:
1 - split
2 - validate length of splitted command
3 - unpack

Reply:
oops,sorry